### PR TITLE
changelog for 5.3

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Version 5.3.0
```
Comment (jmprieur) [inline changelog.txt]: We probably need to create a milestone?

**Review by jmprieur — APPROVED** [review]
LGTM.
Thanks, @jennyf19

File: changelog.txt (hunk 1 above)
Comment (jennyf19) [reply changelog.txt]: let's discuss on monday. i think we just run a new build w/5.2.4

Comment (jennyf19) [reply changelog.txt]: @jmprieur as discussed, 5.2.4

**Pull request merged**